spec/internal/app/controllers/admin/test_results_controller.rb

```diff
@@ -0,0 +1,22 @@
+class Admin::TestResultsController < ApplicationController
+
+  def create
+    test_result = TestResult.new test_result_params
+    test_result.save!
+    render json: test_result, root: true
```

Comment:
it might be worth documenting this, as i think it differs from how you usually write BIP controller methods